lib/tasks/featured_link.rake

```diff
@@ -0,0 +1,20 @@
+namespace :featured_link do
+  desc "Converts any existing featured_links to action links"
+  task convert_to_action_links: :environment do
+    Coronavirus::SubSection.all.each do |sub_section|
+      lines = sub_section.content.split("\n")
+
+      next unless lines.first.starts_with?("[")
+
+      first_line = lines.shift
+      sub_section.action_link_url = first_line.split("(").last.split(")").first
+      content = first_line.split("[").last.split("]").first
+      sub_section.action_link_content = content
+      sub_section.action_link_summary = content
```

Comment:
Graham we should be getting these two values from the content item.

Reply:
I think the loop can be simplified quite a lot.

```
Coronavirus::SubSection.all.each do |sub_section|
  next unless sub_section.action_link_url.present?

  content_id = GdsApi.publishing_api.lookup_content_id(base_path: base_path)
  content_item = GdsApi.publishing_api.get_content(content_id)

  sub_section.action_link_content = content_item["title"]
  sub_section.action_link_content = content_item["summary"]
  sub_section.save!
end
```

Reply:
Updated to get fields from live content_item via publishing-api.